interfaces/Matlab_Toolbox_Revamp/Class/Reactor.m

```diff
@@ -0,0 +1,334 @@
+        end
+        
+        function volume = get.V(r)
+            % Get the volume of the reactor.
+            %
+            % return:
+            %    The volume of the reactor contents at the end of the
+            %    last call to 'advance' or 'step'. Unit: m^3.
+            
+            checklib;
+            volume = calllib(ct, 'reactor_volume', r.id);
+        end          
+        
+        function enthalpy_mass = get.H(r)
+            % Get the mass specific enthalpy of the reactor.
+            %
+            % return:
+            %    The mass specific enthalpy of the reactor contents at the
+            %    end of the last call to 'advance' or 'step'. The enthalpy
+            %    is retrieved from the solution vector. Unit: J/kg. 
+            
+            checklib;
+            enthalpy_mass = calllib(ct, 'reactor_enthalpy_mass', r.id);
+        end        
+        
+        function intEnergy_mass = get.U(r)
+            % Get the mass specific internal energy of the reactor.
+            %
+            % return:
+            %    The mass specific internal energy of the reactor contents 
+            %    at the end of the last call to 'advance' or 'step'. The 
+            %    internal energy is retrieved from the solution vector. 
+            %    Unit: J/kg. 
+            
+            checklib;
+            intEnergy_mass = calllib(ct, 'reactor_intEnergy_mass', r.id);
+        end
+        
+        function yi = massFraction(r, species)
+            % Get the mass fraction of a species.
+            %
+            % parameter species:
+            %    String or one-based integer id of the species.
+            
+            checklib;
+            
+            if ischar(species)
+                k = r.contents.speciesIndex(species) - 1;
+            else k = species - 1;
+            end
+            
+            yi = calllib(ct, 'reactor_massFraction', r.id, k);
+        end
+        
+        function massFractions = get.Y(r)
+            % Get the mass fractions of the reactor.
+            %
+            % return:
+            %    The mass fractions of the reactor contents at the end of 
+            %    the last call to 'advance' or 'step'.
+            
+            checklib;
+            
+            nsp = r.contents.nSpecies;
+            y = zeros(1, nsp);
+            for i = 1:nsp
+                y(i) = r.massFraction(i);
+            end
+        end        
+        
+        %% Reactor set methods
+                
+        function setInitialVolume(r, v0)
+            % Set the initial reactor volume.
+            %
+            % parameter v0:
+            %    Initial volume in m^3.
+            
+            checklib;
+            calllib(ct, 'reactor_setInitialVolume', r.id, v0);
+        end
+
+        function r = set.Mdot(r, MFR)
+            % Set the mass flow rate.
+            %
+            % parameter MFR:
+            %    Mass flow rate in kg/s.
```

Comment:
Please make sure that all docstrings are consistently formatted. In other code you used `:param: ...`. We might need to experiment to see what looks good in the documentation.